src/library/Uno.Material/Styles/Controls/v2/ComboBox.xaml

```diff
@@ -416,14 +418,16 @@
 		</Setter>
 	</Style>
 
+
+
 	<Style x:Key="MaterialComboBoxStyle"
 		   TargetType="ComboBox">
 		<Setter Property="Foreground" Value="{ThemeResource ComboBoxForeground}" />
 		<Setter Property="Background" Value="{ThemeResource ComboBoxBackground}" />
 		<Setter Property="BorderBrush" Value="{ThemeResource ComboBoxBorderBrush}" />
 		<Setter Property="BorderThickness" Value="{ThemeResource ComboBoxBorderThickness}" />
 		<Setter Property="Padding" Value="{ThemeResource ComboBoxPadding}" />
-
+		<Setter Property="TextBoxStyle" Value="{StaticResource MaterialComboBoxTextBoxStyle}" />
```

Comment:
just opening a thread here for the textbox on focus padding/alignment problem

Reply:
At a glance, it seems that the presence of the caret is pushing the element.
Have you tried to have a fixed height on the textbox or its scrollviewer (in textbox template) and set them to align bottom?

Reply:
Fixing the Height on the `TextBox` doesn't make any difference. I'm trying now to set the ScrollViewer (in the textbox template) height to 35 and its VerticalAlignment to bottom (also set scrollviewer bckgd to green). I had to remove the Padding and it looks like:

Windows - looks good


https://github.com/user-attachments/assets/d864aca4-2174-4e29-a441-94f8dbcb9331



Skia 


https://github.com/user-attachments/assets/445a1870-c564-4f70-af52-489642c07396



@Xiaoy312 @kazo0